hashdist/cli/frontend_cli.py

```diff
@@ -123,7 +123,16 @@ class Develop(ProfileFrontendBase):
     """
     Builds a development profile in the HashDist YAML profile spec
     format, at the same location as the profile yaml file, but without
-    the .yaml suffix.
+    the .yaml suffix. You can use the 'target' argument to name it differently
+    than your profile yaml file.
+
+    Example:
+
+        hit develop default.yaml xx
+
+    This creates a throw away developmet profile 'xx', into which you can
```

Comment:
Btw, @cekees and I talked about actually storing the development profile inside the `~/.hashdist/bld` directory as well (or perhaps under a separate directory, but inside `~/.hashdist`, unless configured otherwise by the user), so that `hit` knows about it and can easily load it using `hit load xx` (after #325 is in). In light of this, the sentence "The point of a development profile is that it is NOT built in the HashDist cache, it's built in the target directory." is confusing. What exactly do you mean by "is NOT built in the HashDist cache"? 

The only differences are:

* is mutable (i.e. it's writeable, not read-only) and you are free to install things into it by hand outside of hit
* it doesn't have a hash (since it is mutable)
* nothing can depend on it (since it doesn't have a hash), thus can be safely removed at any time

So perhaps this should be mentioned instead. That it is currently installed directly into the target directory outside of Hashdist is just temporary (if we all agree on our proposal above) --- it will probably just be linked, or perhaps not linked at all, or only if the user requests it, since I think profile activation and deactivation should be done using `hit load` and `hit unload`.